src/components/Header/Header.jsx

```diff
@@ -0,0 +1,42 @@
+import Button from "../shared/Button";
+import RelationBtn from "./RelationBtn";
+import ModifyDocBtns from "./ModifyDocBtns";
+import ModifyViewBtns from "./ModifyViewBtns";
```

Comment:
일단 컴포넌트화를 일부만 하셨는데 description에 목업으로 활용하자고 제안 주셨기 때문에
일단 작성된 부분에 한하여 네이밍 제안드립니다

```suggestion
import RelationButton from "./RelationButton";
import docHandlerButtons from "./docHandlerButtons";
import SwitchViewButtons from "./SwitchViewButtons";
```
ModifyDoc을 docHandler같은 더 큰 개념으로 제안드리는 이유는 중앙에 위치한 그룹이 navigate와 manage(+, -) 이 두 카테고리를 감싸고 있기 때문입니다. 가장 상위 카테고리로, Handler가 어떨지 떠올려봤지만 더 나은 단어나 다른 의견이 생각나시면 제안 부탁드리겠습니다!